sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -131,8 +136,41 @@ internal class ViewAnnotationManagerImpl(
     viewUpdatedListenerSet.remove(listener)
   }
 
+  @AnyThread
+  override fun setViewAnnotationUpdateMode(mode: ViewAnnotationUpdateMode) {
+    renderThread.viewAnnotationMode = mode
+  }
+
+  @AnyThread
+  override fun getViewAnnotationUpdateMode(): ViewAnnotationUpdateMode {
+    return renderThread.viewAnnotationMode
+  }
+
+  /**
+   * First call will be always from Mapbox render thread that will be ALWAYS followed up
```

Comment:
I'd rephrased & simplified a bit, to highlight that
- we will have two calls with the same values from render thread, then from the main thread
- it's needed since scheduling to main thread from c++ is faster